python/wtte/transforms.py

```diff
@@ -2,15 +2,19 @@
 from __future__ import division
 from __future__ import print_function
 
+import sys
```

Comment:
Maybe @dakl knows better? He implemented the try and except first. I'm just trying to avoid py3 incompatibility as much as I can :) 

I know importing `sys` is very common but I think the reason why it should be avoided is because gives you access to the whole system which is typically bad.